ts/Core/Axis/GridAxis.ts

```diff
@@ -910,38 +910,41 @@ class GridAxis {
         const gridOptions = options.grid || {};
         const userLabels = axis.userOptions.labels || {};
 
-        if (axis.horiz) {
-            if (gridOptions.enabled === true) {
-                axis.series.forEach(function (series): void {
-                    series.options.pointRange = 0;
-                });
-            }
+        // Fire this only for the Gantt type chart, #14868.
+        if (axis.options.grid && axis.options.grid.enabled) {
```

Comment:
On line 910, we already have `gridOptions = options.grid || {}`. That means line 914 can probably be optimized to `if (gridOptions.enabled) {`